config/karma/shared.karma.conf.js

```diff
@@ -75,13 +75,19 @@ function getConfig(config) {
     coverageIstanbulReporter: {
       combineBrowserReports: true,
       fixWebpackSourcePaths: true,
+      skipFilesWithNoCoverage: true,
       dir: path.join(process.cwd(), 'coverage'),
       reports: [
         'html',
         'json',
         'lcov',
         'text-summary'
       ],
+      'report-config': {
```

Comment:
This isn't needed for the fix, necessarily, but it puts the html report in a separate folder under `./coverage/html`. It felt cleaner to me.

Reply:
(Removed)